lib/src/ui/user_account_layout.dart

```diff
@@ -24,14 +26,24 @@ import 'user_account_view_version.dart';
 
 class UserAccountLayout extends StatelessWidget {
   static const num _cardMarginTop = 2.25;
-  
+
+  static UpvotyStyle style = UpvotyStyle();
+  static Upvoty upvoty = Upvoty(style: style, pressBack: (context) {
+    Navigator.of(context).pop();
+    UserAccount(
```

Comment:
It should call the presenter through the service using Provider.of<UserAccountService>(context).
Otherwise it will create a new object. Something like this:
```
UserAccountPresenter presenter = Provider.of<UserAccountService>(context).presenter;
presenter.open(context);
```